sonar-plugin-api/src/main/java/org/sonar/api/batch/Sensor.java

```diff
@@ -40,7 +40,9 @@
  * 
  *
  * @since 1.10
+ * @deprecated since 5.6 use {@link org.sonar.api.batch.sensor.Sensor}
  */
+@Deprecated
 @BatchSide
 @ExtensionPoint
 public interface Sensor extends CheckProject {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)